node/service/src/lib.rs

```diff
@@ -593,9 +593,9 @@ where
 				&task_manager.spawn_essential_handle(),
 				config.prometheus_registry(),
 				if experimental_block_import_strategy {
-					None
+					true
 				} else {
-					Some(!dev_service)
+					!dev_service
```

Comment:
This condition can be replaced with `!experimental_block_import_strategy`

Reply:
will work on that thanks again :)

Reply:
Shouldn't this be always `false` in Dev mode ? Or what's the use case ?

Reply:
The behaviour should be the same when using the dev_service. In the past `!dev_service` was probably used to indicate when mocking the relay or using a real relay, it should not be necessary anymore. When `true` is passed, we override the `fork_choice` to `ForkChoiceStrategy::Custom(block_import_params.origin == sp_consensus::BlockOrigin::NetworkInitialSync)`.